### ISSUE TITLE
Is this command a typo error? it causes auto relaunch every time.

### ISSUE THREAD

**Issue by gordonpro** [NathanaelA/nativescript-liveedit#12]
https://github.com/NathanaelA/nativescript-liveedit/blob/e28245f0711de65692c7e747f81813ea00839ef6/support/watcher.js#L406

```
adb shell ps ^| grep '+projectData.nativescript.id
```

Why not

```
adb shell ps | grep '+projectData.nativescript.id
```

**Comment by NathanaelA** [comment]
Good catch; that is actually for windows which is my primary dev environment.  I totally forgot that Linux/Mac might have a different escape character.  Or might not need an escape character.

**Comment by gordonpro** [comment]
I removed the `^` it works fine on OS X.

**Comment by NathanaelA** [comment]
Thanks for the info, I'll get a patched version up shortly.

**Comment by NathanaelA** [comment]
This is fixed in v1.53, thanks for letting me know!
